app/src/main/java/com/github/palFinderTeam/palfinder/easter_egg/ManeameaFragment.kt

```diff
@@ -0,0 +1,44 @@
+package com.github.palFinderTeam.palfinder.easter_egg
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.graphics.BitmapFactory
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.WindowManager
+import android.widget.VideoView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.DialogFragment
+import com.github.palFinderTeam.palfinder.R
+
+
+class ManeameaFragment : DialogFragment(){
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val v: View = inflater.inflate(R.layout.activity_video, container, false)
+        val videoView = v.findViewById<VideoView>(R.id.easter_egg_video)
+        val uri = Uri.parse("android.resource://" + requireActivity().packageName + "/" + R.raw.maneamea)

```

Comment:
We do not have the right for the video and the video should not be in the git

Reply:
So maybe we credit the source somewhere in the project (I found it on ImGur, and this site specifies that you are free to use its content if for non-commercial use).
And where should we store it if not on git ?

Reply:
Git is probably the most convenient place to put it even if it is not really meant for it, and if it's free to use we shouldn't care too much I guess